docs/developer/decision-records/2022-06-28-entities-timestamp/README.md

```diff
@@ -0,0 +1,169 @@
+# Entities timestamp
+
+## Decision
+
+An entity represents any singular, identifiable and separate object inside an application.
+
+The decision based on the definition explained above is to add a timestamp to the EDC entities in their state of
+creation. These Entities are `ContractAgreement`, `ContractDefinition`, `PolicyDefinition`, and `Asset`.
+
+The entity classes `ContractNegotiation` and `TransferProcess` will extend the `StatefulEntity` class. The other
+entities will extend the `Entity` class. the `Entity` class will take the attributes `id` and `createdTimestamp` from
```

Comment:
I would use the refactoring ubiquitous language, saying that the superclass `Entity` is extracted from `StatefulEntity` and the fields `id` and `createdTimestamp` are pulled up to it.